moped-editor/src/components/FileUpload/FileUpload.js

```diff
@@ -24,12 +24,11 @@ registerPlugin(
 
 const FileUpload = (props) => {
   const theme = useTheme();
+  const { getCognitoSession } = useUser();
```

Comment:
i didn't want to make an even larger and more confusing diff, but do y'all think that `useAuthentication` would be a better name for this hook? It does more than get the user session.

Reply:
thanks! i am going to punt on this and put it in https://github.com/cityofaustin/atd-data-tech/issues/24261 for later refinement.